### ISSUE TITLE
[Feature Request] Persisting cache and retreive data to disk

### ISSUE THREAD

**Issue by vzool** [muesli/cache2go#32]
Hi, I like this library because it is simple to use. But, I couldn't find a way to persisting cache to disk in order to retrieve cache after accidental application shutdown. Do you like to add this feature in the future?

Thanks

**Comment by chengyayu** [comment]
why don't you use Redis? I think this package is best used in the business layer cache.

**Comment by chengyayu** [comment]
> Hi, I like this library because it is simple to use. But, I couldn't find a way to persisting cache to disk in order to retrieve cache after accidental application shutdown. Do you like to add this feature in the future?
> 
> Thanks

why don't you use Redis? I think this package is best used in the business layer cache.

**Comment by vzool** [comment]
Yes, I think so. But, I prefer a simple solution and I just did it by swap this library with **LevelDB** as a cache.
Thanks for your response ^_^

**Comment by muesli** [comment]
I tend to agree: I think this is out of cache2go's scope. That said it would probably be fairly simple to (un)marshall the data, and I'm not opposed to a patch - should someone fancy tackling it.
